src/test/ui/consts/projection_qualif.rs

```diff
@@ -7,7 +7,7 @@ use std::cell::Cell;
 const FOO: &u32 = {
```

Comment:
FWIW, I do not understand the filename of this test... but that is pre-existing.